.github/static/DockerQt

```diff
@@ -24,16 +24,16 @@ RUN chmod +x llvm.sh
 RUN ./llvm.sh 20
 
 WORKDIR /
-RUN wget https://download.qt.io/official_releases/qt/6.10/$QT_VERSION/single/qt-everywhere-src-$QT_VERSION.tar.xz
+RUN wget https://download.qt.io/official_releases/qt/6.8/$QT_VERSION/single/qt-everywhere-src-$QT_VERSION.tar.xz
```

Comment:
`6.8` can be inferred from `$QT_VERSION`, through `sed` or `awk` etc, so we do not need to touch too many places - just for changing a single version number :)